src/config.py

```diff
@@ -37,9 +37,13 @@
     OS_VERSION = os.popen(f'{powershell_path} {version_path}').read().split(' ')[-1].replace('\n', '')
 
 else:
-    # OS_VERSION example: '20.04.1'
-    version_pattern = re.compile('~(.*)-')
-    OS_VERSION = re.search(pattern=version_pattern, string=platform.uname().version).group(1)
+    # OS_VERSION example: '20.04'
+    release_lines = open('/etc/os-release').readlines()
+    release_info = {
+        k.lower(): v.replace('"', '').replace('\n', '')
+        for k, v in [line.split('=') for line in release_lines]
+    }
+    OS_VERSION = release_info.get('version_id')
```

Comment:
From the link:
```
       If you are reading this file from C code or a shell script to determine the OS or a
       specific version of it, use the ID and VERSION_ID fields, possibly with ID_LIKE as
       fallback for ID. When looking for an OS identification string for presentation to the user
       use the PRETTY_NAME field.

       Note that operating system vendors may choose not to provide version information, for
       example to accommodate for rolling releases. In this case, VERSION and VERSION_ID may be
       unset. Applications should not rely on these fields to be set.

       Operating system vendors may extend the file format and introduce new fields. It is highly
       recommended to prefix new fields with an OS specific name in order to avoid name clashes.
       Applications reading this file must ignore unknown fields. Example:
       "DEBIAN_BTS="debbugs://bugs.debian.org/""
```
It's the best shot ;)